src/tools/namespace_tool.cpp

```diff
@@ -122,8 +123,10 @@ int NameSpaceTool::RunCommand(const std::string &cmd) {
             return 0;
         }
     } else if (cmd == kCreateCmd) {
+        bool normalFile = !FLAGS_isDir;
         return core_->CreateFile(fileName, FLAGS_fileLength * mds::kGB,
-                             FLAGS_stripeUnit, FLAGS_stripeCount);
+                                 normalFile, FLAGS_stripeUnit,
```

Comment:
IMO normalFile --> !FLAGS_isDir is better

Reply:
In order to be consistent with the parameters of the function being called.

Reply:
> In order to be consistent with the parameters of the function being called.

It would be better to use isFile for Flag, and the meaning of this parameter is indicated in the comment of the function.